src/main/modules/api/cos-api.ts

```diff
@@ -0,0 +1,19 @@
+import { AxiosInstance } from 'axios';
+import { LoggerInstance } from 'winston';
+
+export class COSApi {
+  constructor(
+    private readonly axios: AxiosInstance,
+    private readonly logger: LoggerInstance
+  ) { }
+
+  public createCase(data: any): Promise<boolean> {
+    return this.axios
+      .post('/submit', data)
+      .then(results => !!results)
```

Comment:
I assume this is just temporary?

Reply:
Again, wasnt sure what is being returned so i left it as, if theres data, there is a case.